tests/e2e/c/dynamic_fees.go

```diff
@@ -42,14 +42,14 @@ var _ = e2e.DescribeCChain("[Dynamic Fees]", func() {
 		privateNetwork := e2e.Env.NewPrivateNetwork()
 
 		ginkgo.By("allocating a pre-funded key")
-		key := privateNetwork.GetConfig().PreFundedKeys[0]
+		key := privateNetwork.PreFundedKeys[0]
 		ethAddress := evm.GetEthAddress(key)
 
 		ginkgo.By("initializing a coreth client")
-		node := privateNetwork.GetNodes()[0]
+		node := privateNetwork.Nodes[0]
 		nodeURI := tmpnet.NodeURI{
-			NodeID: node.GetID(),
-			URI:    node.GetProcessContext().URI,
+			ID:  node.ID,
```

Comment:
optional nit: I think using NodeID is more explicit than ID, which can be confusing at times, so I'd prefer to keep the field name `NodeID` here